diffkemp/simpll/DifferentialFunctionComparator.cpp

```diff
@@ -80,11 +81,14 @@ int DifferentialFunctionComparator::cmpValuesByMapping(const Value *L,
 /// Uses data saved in StructFieldNames.
 int DifferentialFunctionComparator::cmpGEPs(const GEPOperator *GEPL,
                                             const GEPOperator *GEPR) const {
+    PREP_LOG(<< getDebugIndent() << "L GEP: " << *GEPL << "\n"
+             << getDebugIndent() << "R GEP: " << *GEPR << "\n");
```

Comment:
I'm wondering if we can get rid of the leading `<< getDebugIndent() <<` as in `LOG`. I assume that doing:
```
PREP_LOG("L GEP: " << *GEPL << "\n");
PREP_LOG("R GEP: " << *GEPR << "\n");
```
is not feasible b/c `prep_context` would be called twice.
Could we achieve this?
```
LOG_PREP("L GEP: " << *GEPL << "\n", "R GEP: " << *GEPR << "\n");
```
or maybe this?
```
LOG_PREP("L GEP: " << *GEPL << "\n" << "R GEP: " << *GEPR << "\n")
```
and the macro would magically apply the indent at the beginning of each line?

Just throwing ideas to make the usage simpler, feel free to comment them.

Reply:
I've opted for the last option and added a function which automatically adds debug indent to each newline (except for the last one). The current implementation is a bit non-optimal in the way it handles the strings, if that's a problem I could re-do it a better way. 

I'd also like to just point out one small thing, including the leading << in the macro and not in the macro call leads to ugly alignment of the call in the code. I can't cite the source code from an outdated comment but basically this:

```
    PREP_LOG("L GEP: " << *GEPL << "\n"
                       << "R GEP: " << *GEPR << "\n");
```

Reply:
Ok, this is annoying but for the sake of consistency (avoiding leading `<<`), I guess that we don't have another option. In addition, since we'll eventually store just the pointers, this will likely change.